ludwig/modules/metric_modules.py

```diff
@@ -123,11 +123,27 @@ def get_inputs(cls):
 
 
 @register_metric(ROC_AUC, [BINARY])
-class ROCAUCMetric(AUROC, LudwigMetric):
-    def __init__(self, **kwargs):
-        super().__init__(dist_sync_fn=gather_all_tensors)
+class ROCAUCMetric(LudwigMetric):
+    """Fast implementation of metric for area under ROC curve."""
 
-    """ Metric for area under ROC curve. """
+    def __init__(
+        self,
+        num_thresholds: int = 201,
+        epsilon: float = 1e-7,
+        compute_on_step: Optional[bool] = None,
```

Comment:
The generic torchmetrics params should be capable of being passed through the `**kwargs`, so shouldn't need to specify explicitly.

Reply:
Updated the code to follow a similar pattern as what the other metrics are doing. It looks like we pass in some common arguments to each metric module in **kwargs, so I'm not passing in all kwargs to the torchmetrics constructor.